src/constants/index.ts

```diff
@@ -1,164 +1,32 @@
-    case 5:
-      return utils.getAddress("0x287094e10A70DEecfD8bC377317bf3e9FE4bD287");
-    case 42:
-      return utils.getAddress("0xc6791fDab00944E18Ee809FC54a21B31f8A0B612");
-    case 56:
-      return utils.getAddress("0x2caD01dE572e00af828e5FB02EBf9821f03eD8f0");
-    case 100:
-      return utils.getAddress("0x2caD01dE572e00af828e5FB02EBf9821f03eD8f0");
-    case 137:
-      return utils.getAddress("0x0d738c7fd96717305d82E5EF1D0fcc45C71042Fa");
-    case 1284:
-      return utils.getAddress("0x34670AF75F42efcDcC0B946006a3b3a2E2cD61B2");
-    case 1285:
-      return utils.getAddress("0x5c54A5C104f90f5F987FeE09e3C579F40AC43301");
-    case 9001:
-      return utils.getAddress("0x38306FB998e48b3e4BCE9AB84463aae536413Cc4");
-    case 43114:
-      return utils.getAddress("0x69B910a1B856193784a7aa65F57830301D296Ea1");
-    case 44787:
-      return utils.getAddress("0x0d738c7fd96717305d82E5EF1D0fcc45C71042Fa");
-    case 80001:
-      return utils.getAddress("0x832e030fd726a032D4Bd1A6922363c8aa3cB4fFD");
-    default:
-      throw new Error(
-        `getMetaBoxPullFeeAddress: chainId ${chainId} not supported`
-      );
-  }
-};
-
-const getRelayForwarderAddress = (chainId: number): string => {
+export const getRelayAddress = (chainId: number): string => {
   switch (chainId) {
-    case 1:
-      return utils.getAddress("0x5ca448e53e77499222741DcB6B3c959Fa829dAf2");
-    case 4:
-      return utils.getAddress("0x9B79b798563e538cc326D03696B3Be38b971D282");
-    case 5:
-      return utils.getAddress("0x61BF11e6641C289d4DA1D59dC3E03E15D2BA971c");
-    case 42:
-      return utils.getAddress("0x4F36f93F58d36DcbC1E60b9bdBE213482285C482");
-    case 56:
-      return utils.getAddress("0xeeea839E2435873adA11d5dD4CAE6032742C0445");
-    case 100:
-      return utils.getAddress("0xeeea839E2435873adA11d5dD4CAE6032742C0445");
-    case 137:
-      return utils.getAddress("0xc2336e796F77E4E57b6630b6dEdb01f5EE82383e");
-    case 1284:
-      return utils.getAddress("0x14cdD6d9eBfbB7DAAF09395E56B2A89905D62b4C");
-    case 1285:
-      return utils.getAddress("0x79A0cB573D3Db184752511969F1b869A184EA445");
-    case 9001:
-      return utils.getAddress("0x9561aCdf04C2B639dFfeCB357438e7B3eD979C5C");
-    case 43114:
-      return utils.getAddress("0x3456E168d2D7271847808463D6D383D079Bd5Eaa");
-    case 44787:
-      return utils.getAddress("0xc2336e796F77E4E57b6630b6dEdb01f5EE82383e");
-    case 80001:
-      return utils.getAddress("0x3428E19A01E40333D5D51465A08476b8F61B86f3");
+    case 80_001:
+      return utils.getAddress("0xaBcC9b596420A9E9172FD5938620E265a0f9Df92");
     default:
-      throw new Error(
-        `getRelayForwarderAddress: chainId ${chainId} not supported`
-      );
+      throw new Error(`getRelayAddress: chainId [${chainId}] is not supported`);
   }
 };
 
-const getRelayForwarderPullFeeAddress = (chainId: number): string => {
+export const getRelayWithTransferFromAddress = (chainId: number): string => {
```

Comment:
I think it's more robust to have all chain config centralized in a typed object, so that you are sure that when you add a new network you don't forget to add one piece of config somewhere.

can have a look at the "address book" pattern that we are using on some other projects:
https://github.com/gelatodigital/ops-sdk/blob/master/src/constants/index.ts

Reply:
https://github.com/gelatodigital/relay-sdk/pull/3/commits/6d839babd52e19b77ea4b96c75b0351f9b3407c0